Rebus.AdoNet/Dialects/SqliteDialect.cs

```diff
@@ -1,70 +1,70 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Data;
-using System.Data.Common;
-using System.Linq;
-using System.Text;
-
-namespace Rebus.AdoNet.Dialects
-{
-	public class SqliteDialect : SqlDialect
-	{
+﻿using System;
```

Comment:
Please, this file appears to have changed entirely. Fix it so only changed parts appear on this diff.

Reply:
ack